src/openrpc/keyboard.json

```diff
@@ -175,11 +178,9 @@
         {
           "name": "event",
           "x-response": {
-            "$ref": "#/components/schemas/KeyboardResult",
+            "type": "string",
```

Comment:
breaking change?

Reply:
I think I would rather the `use` methods change to an extensible object instead of the provide methods change to a non-extensible string.